### ISSUE TITLE
Insecure HTTP is not allowed by platform

### ISSUE THREAD

**Issue by ghost** [invoiceninja/admin-portal#238]
Hey  
  
I'm trying the app for Android F-Droid v5.  
I've invoice ninja running at home without https. I added `REQUIRE_HTTPS=false` to the .env file which is for invoiceninja itself I guess and has nothing to do with the app?  

I get the following message:  
`Bad state: Insecure HTTP is not allowed by platform: http://localhost:8000/api/v1/login?first_load=true&include_static=true`

Any chance to get access without the certificate?

**Comment by hillelcoren** [comment]
Hi,

I've looked into it, I believe it's possible but requires setting a flag when compiling the app which would affect all users.

https://stackoverflow.com/a/65730723/497368 

We could probably make a special APK available as part of our CI/CD pipeline?

**Comment by DanSman** [comment]
Yeah, I have found that too and I checked the manifest but I couldn't find it. Otherwise I can compile the apk by myself. But I'm not a dev so, I need a little help for that :/

**Comment by mstyp** [comment]
I am also having this problem. I am using a self signed certificate. I cant build the app myself, unless I can do so via copy paste.
